src/WebUI/ClientApp/src/app/app.module.ts

```diff
@@ -40,11 +46,13 @@ import { ChartsModule } from 'ng2-charts';
     ApiAuthorizationModule,
     ChartsModule,
     RouterModule.forRoot([
-      { path: '', component: HomeComponent, pathMatch: 'full' },
+      { path: '', component: DashboardComponent, pathMatch: 'full' },
+      { path: 'Home', component: HomeComponent },
       { path: 'counter', component: CounterComponent },
```

Comment:
ℹ️  you may remove the template routes `todo`, `fetch-data`, `counter`

Reply:
I refer these components all the time. I would like to keep these for few more days.